### PR TITLE
This resolves #11

### LINKED ISSUE #11: Need to update my markdown

### PR DESCRIPTION
**Pull request by derekpeng3**

### CODE REVIEW

**Review by jameskim03 — APPROVED** [review]
Looks good!

**Review by LASC-Pvu — APPROVED** [review]
amazing code!

File: _slides/09-derekpeng3.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-Caption here
+Welcome to Github
```
Comment (makwei88) [inline _slides/09-derekpeng3.md:6]: Good review

**Review by makwei88 — APPROVED** [review]
approve

**Pull request merged**